compiler/circle-opselector/src/OpSelector.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __CIRCLE_OPSELECTOR_OPSELECTOR_H__
+#define __CIRCLE_OPSELECTOR_OPSELECTOR_H__
+
+#include <luci/IR/Module.h>
+
+#include <luci/Importer.h>
+
+#include <vector>
+
+namespace luci
+{
+void post_import_graph(luci::Module *module, const luci::CircleReader &reader);
+}
+
+namespace opselector
+{
+
+class OpSelector
```

Comment:
I think it would be better to revise this PR to introduce this `class OpSelector` and change th comment explaining what it doesn.